spec/support/shared_examples/controllers/unauthenticated_action.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'an unauthenticated action' do
+  it { is_expected.to have_http_status(302) }
```

Comment:
it's better to use symbol status codes https://gist.github.com/mlanett/a31c340b132ddefa9cca